Countries/ListViewController.swift

```diff
@@ -82,7 +92,7 @@ class ListViewController: UITableViewController, UISearchBarDelegate, UISearchRe
         searchController.searchBar.tintColor = .black
     }
 
-    private func registerTableView() {
+    func registerTableView() {
```

Comment:
why `private` was removed?

Reply:
I can't remember why it was removed, so I inserted it again